packages/grafana-ui/src/components/ColorPicker/ColorPickerPopover.tsx

```diff
@@ -116,16 +116,8 @@ class UnThemedColorPickerPopover<T extends CustomPickersDescriptor> extends Comp
         */}
         <div tabIndex={-1} className={styles.colorPickerPopover}>
           <TabsBar>
-            <Tab
-              label={t('grafana-ui.color-picker-popover.palette-tab', 'Colors')}
-              onChangeTab={this.onTabChange('palette')}
-              active={activePicker === 'palette'}
-            />
-            <Tab
-              label={t('grafana-ui.color-picker-popover.spectrum-tab', 'Custom')}
-              onChangeTab={this.onTabChange('spectrum')}
-              active={activePicker === 'spectrum'}
-            />
+            <PaletteTab activePicker={activePicker} onTabChange={this.onTabChange} />
+            <SpectrumTab activePicker={activePicker} onTabChange={this.onTabChange} />
```

Comment:
really minor refactor nit. think an `isActive` prop might make more sense than passing the whole `activePicker` down?

Reply:
Fixed 👍